src/engraving/dom/harmony.h

```diff
@@ -263,6 +235,7 @@ class Harmony final : public TextBase
 
     double mag() const override;
     void setUserMag(double m) { m_userMag = m; }
+    const std::optional<double>& userMag() const { return m_userMag; }
```

Comment:
I love the idea of using modern features, but this seems a bit off-pattern to me. It find it hard to interpret `mag` as something that "may or may not exist". To me, `mag` is something that should just be 1, whenever it hasn't been explicitely set otherwise.